ORBIT/phases/install/oss_install/floating.py

```diff
@@ -189,9 +190,29 @@ def install_floating_substations(
             constraints=vessel.operational_limits,
         )
         yield position_onsite(vessel)
-        yield perform_mooring_site_survey(vessel)
-        yield install_mooring_anchor(vessel, depth, "Suction Pile")
-        yield install_mooring_line(vessel, depth)
+        yield vessel.task_wrapper(
+            "Ballast to Operational Draft",
+            6,
+            constraints={"windspeed": le(15), "waveheight": le(2.5)},
+        )
+
+        for _ in range (3):
+            yield perform_mooring_site_survey(vessel)
+            yield install_mooring_anchor(vessel, depth, "Suction Pile")
+            yield install_mooring_line(vessel, depth)
+            yield vessel.task_wrapper(
+                "Connect Mooring Lines",
+                22,
+                suspendable=True,
+                constraints={"windspeed": le(15), "waveheight": le(2.5)},
+            )
+            yield vessel.task_wrapper(
+                "Check Mooring Lines",
+                12,
+                suspendable=True,
+                constraints={"windspeed": le(15), "waveheight": le(2.5)},
+            )
+
         yield vessel.transit(distance)
 
 
```

Comment:
The SubstationAssemblyLine class looks good for now, but we should review with Aaron or someone - is it really possible to assemble a 5000 tonne topside on a floating platform and tow it out to sea???